src/wallet/wallet2.cpp

```diff
@@ -1929,7 +1934,7 @@ void wallet2::process_outgoing(const crypto::hash &txid, const cryptonote::trans
 //----------------------------------------------------------------------------------------------------
 void wallet2::process_new_blockchain_entry(const cryptonote::block& b, const cryptonote::block_complete_entry& bche, const parsed_block &parsed_block, const crypto::hash& bl_id, uint64_t height, const std::vector<tx_cache_data> &tx_cache_data, size_t tx_cache_data_offset)
 {
-  THROW_WALLET_EXCEPTION_IF(bche.txs.size() + 1 != parsed_block.o_indices.indices.size(), error::wallet_internal_error,
+  THROW_WALLET_EXCEPTION_IF(bche.txs.size()>0 && bche.txs.size() + 1 != parsed_block.o_indices.indices.size(), error::wallet_internal_error,
```

Comment:
that's incorrect. block always has at least miner transaction. that means `parsed_block.o_indices.indices.size()` should always contain at least outputs for miner transaction.

If it will cause problems for fastsync it may be wrapped in `if` block from outside

Reply:
ok will remove it